.github/workflows/rust-audit.yml

```diff
@@ -0,0 +1,22 @@
+name: Rust audit
+
+on:
+  schedule:
+    - cron: '0 2 * * *' # run at 2 AM UTC
+  push:
+    branches: [ master ]
+  pull_request:
+    branches: [ master ]
+
+jobs:
+  build:
+    runs-on: ubuntu-latest
+    if: "!contains(github.event.head_commit.message, 'ci skip')"
```

Comment:
Maybe this should be `audit skip` instead.

Reply:
That would introduce another term to have in the commit messages, I see some value in having that granularity, but isn't it a bit to granular? Who would ever use that commit message?

Reply:
yeah, seems unnecessary.